client/asset/dcr/spv.go

```diff
@@ -1030,6 +1031,11 @@ func newVSPClient(w vspclient.Wallet, vspHost, vspPubKey string, log dex.Logger)
 	}, log)
 }
 
+// Rescan performs a blocking rescan, sending updates on the channel.
+func (w *spvWallet) Rescan(ctx context.Context, c chan wallet.RescanProgress) {
+	w.dcrWallet.RescanProgressFromHeight(ctx, w.spv, 0, c)
```

Comment:
Should use the new seed's encoded birthday time.

Reply:
I've removed birthday configuration from spv wallets altogether and plumbed through the seed birthday where it's needed. This does mean that wallets that use an old seed for a rescan or  wallet recovery will use the default wallet birthday from 2021. This actually doesn't matter to recovery of a dcr wallet, since dcr always scans the full blockchain for the initial scan. **dcrwallet** actually has no concept of a birthday, from what I can tell. But we still use the seed birthday for limiting the scope of a rescan.

Reply:
You can rescan from a birthday in the next release. https://github.com/decred/dcrwallet/commit/5e8f1328a4dd5360a75c4de746f5384cc12aa921